### PR TITLE
docs: 기존 머지 PR 중 발견 된 용어 불일치 수정

### PR DESCRIPTION
**Pull request by rltq**
## 라이선스 동의
- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
간단한 내용 수정이므로 이슈 발행 없이 PR을 남깁니다.

## PR 종류
- [x] 오탈자를 수정하거나 번역을 개선하는 기여
- [ ] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명

[TRANSLATION_GUIDE](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md) 를 반영하여 PR 800번대 까지의 용어 불일치를 일부 정정합니다. 기입되어 있는 PR 번호는 참고용이며, 원 번역자의 작업을 존중하는 방향으로 최소 정정했습니다. 인용하게 될 PR 이 다수이므로 태그를 넣지는 않았습니다. 필요하시다면 # 추가하여 PR 연결하겠습니다.

#### 원문: The first step for pruning the model is to insert parametrizations for# masking the weights of the model
- 기존: **모델의 매개변수화는 첫 번째 단계는** 모델의 가중치를 마스킹하기 위한 매개변수화를 삽입하는 것입니다. (PR 927)
- 변경: **모델의 가지치기를 위한 첫번째 단계는** 모델의 가중치를 마스킹하기 위한 매개변수화를 삽입하는 것입니다.
- 비고: `pruning` 용어를 살려 번역해야 의미가 누락되지 않습니다.

#### 용어 통일
- (PR 879) "레이어의 출력" 표기를 '계층의 출력'으로 정정. 원문에 따라 '출력' 역시 output으로 번역해야 하나 원본 번역을 존중하여 의역. 그 외 "레이어" 관련 교정 4 건 "파라미터" 관련 교정 2 건
- (PR 945) "파라미터의 변화도" 표기를 '매개변수의 변화도'로 정정
- (PR 1052) 다수의 '레이어' 표기를 '계층'으로 정정 -> 필요. merge 전
- (PR 923) "드롭아웃 레이어" 를 'Dropout 계층'으로 정정 -> 확인 결과 해당 PR 외에도 Dropout으로 표기된 건이 10개 이내 존재. PR을 모두 추적하여 근거를 달기 어려운 상태이므로 923 정정 건만 수정
- (PR 879) parameter의 "인수" 표기를 '인자'로 정정 -> 확인 결과 인자 역시 해당 PR 외 인수로 표기된 건이 10개 이내 존재
- (PR 869) confusion matrix의 "오차 행렬" 표기를 '혼동 행렬'로 정정, 영문 병기 유지. 그 외 plotting의 "설정" 표기를 '도식화'로 정정 1건
- (PR 869) gradient의 "기울기" 표기를 '변화도'로 정정

#### 오타 정정. 
- (PR 927) "메서드" 를 가이드라인에 따라 "메소드"로 정정 

---

### 그 외 혼용 표기

아래와 같은 항목에서 혼용 표기가 다수 발견되었습니다. TRNSLATION_GUIDE 수정 또는 단순 단어 치환(작업량 S)이 가능할 것으로 보입니다. PR 내용에서 벗어나 별도의 항목에 기술합니다.

#### `training` (학습) 용어 불일치: 대체로 훈련과 혼용되어 표기됨. 
- 1040 ("훈련합니다")
- 961 ("훈련 함수") 
- 959 ("훈련 함수")
- 945 ("훈련을 재개하려면", "훈련을 재개하고 싶다면")
- 869 ("훈련이 아님")

동일 조직의 terms.kr 에서 진행하는 표준화 작업에 의하면 해당 용어는 훈련으로 번역되며, 개인적인 의견으로도 learning과의 구분을 위해 훈련으로 번역하는 것이 혼동 방지에 도움이 될 것 같습니다. 

#### `layer` (계층) 용어 불일치: 레이어로 번역되고 있음
- 879 ("레이어" "선형 레이어" "대규모 레이어의 출력", "레이어와 파라미터 순서") 외 다수

로그를 살펴보면 800번대에서 '레이어' 로 암묵 번역된 이후로 다수의 번역에서 레이어라는 표현을 사용하고 있습니다.  (코드베이스 서치 결과 76개 파일, html 등의 파일 포함이므로 실제는 이보다 적을 것으로 예상) 다만 계층이라는 표현 역시 다수 사용되고 있으므로 통일 또는 TRANSLATION_GUIDE 업데이트가 필요해보입니다.

### `parameter` (매개변수) 용어 불일치: 파라미터와 혼용
- 1052 ("온도 파라미터") - 아직 Open 된 PR
- 945 ("파라미터의 변화도")

마찬가지로 관습상 사용되는 파라미터 음차와 매개변수라는 번역어가 혼용되고 있습니다.

그 외: `shape` 와 `output`은 의역으로 변형되어 번역되고 있습니다. `argument` 역시 인자라는 번역어가 지정되어 있음에도 인수 등으로 혼용되고 있습니다.

### CODE REVIEW

**Comment by rltq** [comment]
아! 기여 문서를 분명 읽었는데 이슈 선행 제출 조건을 미처 보지 못했습니다. 불필요한 PR이라면 편히 닫아주세요. 감사합니다. 🙏

**Review by hyoyoung — CHANGES_REQUESTED** [review]
수정 사항은 잘되어있는 것으로 보입니다.
다음과 같은 부분은 개선 부탁드립니다.

1. 이슈 추가
2. 커밋 1개에는 1개의 의미만 담아주세요
지금은 indent fix와 번역 개선이 같이 있고
번역 개선도 여러가지가 섞여있습니다.
개별로 분리 부탁드립니다

**Comment by rltq** [comment]
검토 감사합니다. 확인 된 번역어 혼용사례 가이드에 맞게 정정했습니다. 
하나 질문드리고 싶은 점은: 이슈 템플릿을 보면 수정이 필요한 문서 URL을 기입하게 되어있고 변경사항을 기재하게 되어있는데 어디까지 작성해야 할까요? 

문서 기준으로 하면 변경된 번역어가 섞여서 기재될 것 같고 커밋처럼 변경 단어 단위로 기재하면 이슈에서 검토하시기 어려울 것 같습니다. 편하신쪽으로 이슈 발행하겠습니다.

기존 PR에 작성한 번역 개선 건은 따로 분리하겠으니 이번엔 번역된 용어가 가이드라인에 맞는지만 확인 부탁드립니다.
언제나 감사합니다.

**Comment by hyoyoung** [comment]
이럴 경우에는 이슈는 1개로 적어도 좋을거 같습니다.
다만 그 경우에도 PR은 개별 단어끼리 묶어서 보내주시는게 리뷰가 더 수월할거 같습니다
이를테면 매개변수 PR, 학습 PR 이렇게 되어있는게 좋을거 같습니다

**Comment by rltq** [comment]
#1133 

말씀하신대로 이슈에서 정정이 필요한 항목을 확인할 수 있게 목록화했습니다. 정정 내용을 일일이 링크로 붙이는건 리뷰하는 입장에서도 번거로운 일일 것 같아 정정이 필요한 이유를 대신 기입해두었습니다. 상세 내용은 diff로 확인하시는 게 좋지 않을까 합니다. 목록이 많긴 하지만 단순히 용어 교체정도에 머무르는 작업이라 간단히 읽어보시는 것으로도 충분할 듯 합니다.

PR을 올리기 전에 하나 먼저 확인 요청드리고 싶은데, 위와 같은 이슈대로 PR을 올리면 총 9여 건이 되는데 이 점 괜찮으실까요? 중요한 풀리퀘가 묻힐까 염려됩니다. 브랜치를 분할하는 등의 제 쪽에서 할 수 있는 작업은 끝내두었지만, 노고가 많으신 점을 알기에 불필요한 일을 줄이고자 생성 전에 질문드립니다.

우선 확인을 요청드리고자 PR 2개를 미리 올려두었습니다. 이슈 하단에서 확인해주시면 됩니다. 

감사합니다.

@hyoyoung
